### ISSUE TITLE
[Accessibility] Narrator cannot announce the expand/collapse state for treeView with checkbox enabled when expand/collapse any treeNode

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#6732]
* .NET Core Version: 
7.0.100-preview.2.22116.19

* Have you experienced this same bug with .NET Framework?: 
Yes

**Minimal repro:**
1. Create a Winforms .Net Core application with treeView control added
2. In treeView, add several root treeNodes and child treeNodes

Or use following attached application:
[WinFormsApp4.zip](https://github.com/dotnet/winforms/files/8107369/WinFormsApp4.zip)

3. Build and run this application
4. Launch Narrator to test expand/collapse root treeNode

**Actual result:**
Narrator cannot announce the expand/collapse state for treeview nodes when expanding or collapsing them. 
![image](https://user-images.githubusercontent.com/26474449/154913528-20292719-0c2c-4dca-8880-8a68325a36dc.png)
![TreeView_CheckBox](https://user-images.githubusercontent.com/26474449/154913766-afd3b6f7-cc49-41f1-9e1e-511b52930d85.gif)

**Expected result:**
Narrator can announce the expand/collapse state for treeview nodes when expanding or collapsing them

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang  - what do you mean here -  `when doing expanding or collapsing action for treenodes.` - how do you do this action, do you invoke it in inspect?

**Comment by Amy-Li03** [comment]
Verified this issue on .NET 7.0.100-preview.4.22227.3, issue was fixed. Narrator can announce the expand/collapse state when focusing treeview checkbox nodes or doing expanding or collapsing action for them.
![image](https://user-images.githubusercontent.com/62929087/165693014-69f412fb-348d-4091-b55a-233b87391f48.png)

**Comment by Ashley-Li** [comment]
Verified this issue on .NET 7.0.100-preview.5.22227.7, issue was fixed. Narrator can announce the expand/collapse state when focusing treeview checkbox nodes or doing expanding or collapsing action for them.
![image](https://user-images.githubusercontent.com/56664234/165701341-5805e000-7ae9-454d-9119-ebdb9b09e53c.png)
